### DISCUSSION TITLE
From Meson version 1.2.0 and newer what is the best way to enable C++20 modules

### DISCUSSION THREAD

**Discussion by dreamer-coding** [mesonbuild/meson#13200] [Q&A] [answered]
What is the recommended approach, starting from Meson version 1.2.0 onwards, to enable support for C++20 modules within a C++ project?

**Comment by eli-schwartz** [comment]
#5024 seems relevant.

**Reply by eli-schwartz** [reply, marked as answer]
Note that for basic support, meson will recognize MSVC *.ixx files, and for GCC you can add `cpp_args: ['-fmodules-ts']`.

**Reply by dreamer-coding** [reply]
Seems a bit risky at the moment to add this so I'll wait.

**Comment by harryjwright** [comment]
Any updates on this?

**Comment by dreamer-coding** [comment]
> Any updates on this?

Some significant updates.
 
#5024
